### ISSUE TITLE
LargeTexture.get_data() always returns null

### ISSUE THREAD

**Issue by pfertyk** [godotengine/godot#22457]
**Godot version:** v3.0.6.stable.official.8314054

**OS/device including version:** Linux Pop!OS 17.10 (artful)

**Issue description:**

I've created a LargeTexture instance and added 2 textures to it. Then I've tried to set this texture as camera's environment panorama. I got a black panorama instead. When I've set a StreamTexture (`load()`) it worked fine.

**Steps to reproduce:**
Here is a code sample from my project (scene with a camera, method `_ready`):
```
	var lt = LargeTexture.new()
	lt.set_size(Vector2(1000, 1000))
	lt.add_piece(Vector2(0, 0), load("res://assets/images/space.jpg"))
	lt.add_piece(Vector2(0, 0), load("res://assets/images/planet_transparent.png"))
	print(lt.get_piece_count())
	print(lt.get_size())
	print(lt.get_data())
	print(lt)
	camera.environment.background_sky.panorama = lt
#	this works
#	camera.environment.background_sky.panorama = load("res://assets/images/planet_transparent.png")
	var tex = camera.environment.background_sky.get_panorama()
	print(tex)
```

Output:
```
2
(1000, 1000)
[Object:null]
[LargeTexture:1007]
[LargeTexture:1007]
```

Note: camera uses its own environment (overwriting the default environment of the project). As you can see, `lt.get_data()` returns `null`, but that was not a problem before (I've created another project, where I use the same setup for LargeTexture and use it for a TextureRect; there I've also got a `null`, but the texture worked as expected).

**Minimal reproduction project:**

**Comment by KoBeWi** [comment]
Can anyone still reproduce this bug in [Godot 3.2.2](https://godotengine.org/article/maintenance-release-godot-3-2-2) or any later release?

If yes, please ensure that an up-to-date Minimal Reproduction Project (MRP) is included in this report (a MRP is a zipped Godot project with the minimal elements necessary to reliably trigger the bug). You can upload ZIP files in an issue comment with a drag and drop.

**Comment by pfertyk** [comment]
I forgot that I submitted this one ;) I remember that eventually I've found a workaround, which later turned out to be a correct solution (my experiments with merging textures would produce a wrong result).

I will try to recreate this in the next couple of days. If you don't hear back from me, consider it fixed ;)

**Comment by miskatonicstudio** [comment]
Unfortunately, it still doesn't work, just checked on Godot v3.2.2.stable.official on Linux. I think I figured out a simpler example:

```
	var lt = LargeTexture.new()
	lt.set_size(Vector2(2048, 2048))
	lt.add_piece(Vector2(0, 0), load("res://color.png"))
	var image = lt.get_data()
#	var image = lt.get_piece_texture(0).get_data()  # this works
	image.save_png("texture.png")
```
This causes `Attempt to call function 'save_png' in base 'null instance' on a null instance`. The problem seems to be that `get_data` on LargeTexture always returns null.

**Comment by akien-mga** [comment]
We were discussing this with @reduz but it's unclear what use case `LargeTexture` still has in Godot 3.x. Is there a specific reason why you were attempting to use it instead of a plain `StreamTexture`?

Removing `LargeTexture` would be a simple way to solve this for 4.0 :)

**Comment by miskatonicstudio** [comment]
The idea here was to combine several textures into a single one. While eventually I figured out that this is not a correct solution (instead, I should create a 3D sprite with the smaller texture), the problem in general remains. I've used `LargeTexture` because it's the only one that had `add_piece` method which, as far as I know, adds another texture to the current one. `StreamTexture` doesn't have this method, so it didn't seem the right choice for this job ;)

Is there another way to combine several textures in Godot? If the method `add_piece` is there, it should work, but last time I checked it didn't (as described above). If `LargeTexture` will be removed in Godot 4.0, then what should I use for combining textures?
